sdk/core/core-client-rest/test/browser/streams.spec.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert } from "chai";
+import { getClient } from "../../src/getClient";
+import sinon from "sinon";
+
+function createResponse(statusCode: number, body = ""): Response {
```

Comment:
maybe file an issue to add a test-util for this sort of thing?

Reply:
Yeah that makes sense I filed: https://github.com/Azure/azure-sdk-for-js/issues/21350